### PR TITLE
Clarifies wording for ERC-2

### PR DESCRIPTION
**Pull request by khushal08**
# Overview/Summary

The wording for ERC-2 was not clearly stated and caused confusion amongst the reader. This change is to make the recommendation clear. 

## This PR fixes/adds/changes/removes

1. https://dev.azure.com/CSUSolEng/Azure%20Infra%20Networking/_workitems/edit/32666/

## As part of this Pull Request I have

- [x] Read the [Contribution Guide](https://azure.github.io/Azure-Proactive-Resiliency-Library/contributing) and ensured this PR is compliant with the guide
- [x] Checked for duplicate [Pull Requests](https://github.com/Azure/Azure-Proactive-Resiliency-Library/pulls)
- [x] Associated it with relevant [GitHub Issues](https://github.com/Azure/Azure-Proactive-Resiliency-Library/issues) or ADO Work Items (Internal Only)
- [x] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Azure-Proactive-Resiliency-Library/tree/main)
- [x] Ensured PR tests are passing
- [x] Performed testing and provided evidence (e.g. screenshot of output) for any changes associated to ARG queries and/or scripts
- [x] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

**Review by ejhenry — DISMISSED** [review]
LGTM 👍

**Comment by ehaslett** [comment]
I have been asked to review - please wait to merge until I can sign off on this. Thanks!
https://dev.azure.com/CSUSolEng/Azure%20Infra%20Networking/_workitems/edit/32693

File: docs/content/services/networking/expressroute-circuits/_index.md
Comment (ehaslett) [inline docs/content/services/networking/expressroute-circuits/_index.md:1]: A couple typo related updates recommended:
- Line 66: "Ensure you provider" -> "Ensure your provider"
- Line 68: Expressroute -> ExpressRoute
- Line 68: connection -> connectivity
- Line 68: "from customer edge" -> "from the customer edge"

Comment (khushal08) [reply docs/content/services/networking/expressroute-circuits/_index.md:1]: Done and pushed the changes.

**Comment by ehaslett** [comment]
LGTM
#sign-off

**Review by ehaslett — APPROVED** [review]

**Pull request merged**